packages/@coorpacademy-components/src/molecule/certification-card/index.js

```diff
@@ -44,7 +44,9 @@ const CertificationCard = (props, context) => {
       <div className={style.detailWrapper}>
         <div className={style.titleWrapper}>
```

Comment:
💭  Maybe we could add a dynamic data name:  `certification-$ref` or `certification-$name`